src/templates/blog-post.js

```diff
@@ -16,17 +16,9 @@ function BlogPostTemplate({ data, pageContext: { previous, next }, location }) {
   return (
     <Layout location={location} title={siteTitle}>
       <SEO title={post.frontmatter.title} description={post.excerpt} />
-      <h1
-        style={{
-          marginTop: rhythm(1),
-          marginBottom: 0,
-        }}
-      >
-        {post.frontmatter.title}
-      </h1>
+      <h1>{post.frontmatter.title}</h1>
       <p
         style={{
-          ...scale(-1 / 5),
```

Comment:
I added this so that the date text would be smaller. If you want to have it the same size, that’s fine with me.

Reply:
The date is big enough on the main page.. so why not the same here.

Reply:
Also we should add the author soon.. which probably will go into this section as well..